lib/services/keyVault/lib/keyVault.js

```diff
@@ -1293,7 +1289,7 @@ KeyVaultClient.prototype.deleteCertificateIssuer = function (vaultBaseUrl, issue
  * the issuer of the X509 component of a certificate.
  * 
  * @param {string} [options.certificatePolicy.issuerReference.name] Name of
- * the referenced issuer object.
+ * the referenced issuer object e.g. 'Self', 'Unknown'.
```

Comment:
>the referenced issuer object e.g. 'Self', 'Unknown'. [](start = 3, length = 52)

Either name of the referenced issuer object or one of the reserved name of 'Self' or 'Unknown'